utopia-remix/app/routes/projects.tsx

```diff
@@ -45,12 +90,9 @@ const ProjectsPage = React.memo(() => {
     [],
   )
 
-  const openProject = React.useCallback(
-    (projectId: string) => () => {
-      window.open(`${window.ENV.EDITOR_URL}/p/${projectId}`, '_blank')
-    },
-    [],
-  )
+  const createNewProject = () => {
+    window.open(`/project/`, '_self')
```

Comment:
Because this is not living on the same root as the editor, you'll need to explicitly pass the editor location in front of the path - this is done in another part of the code, it would definitely deserve a helper function tho (🙈)

```suggestion
    window.open(`${window.ENV.EDITOR_URL}/project/`, '_blank')
```